content/yaml/distribution.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Publishing
+description: Code signing and publishing with YAML.
+weight: 8
+---
+
```

Comment:
This doc lacks a proper intro. Also, there's no information about Android code signing here. For iOS, it would make sense here to describe that we use Codemagic CLI tools to perform code signing and publishing for iOS apps (not sure if some part of Codemagic CLI tools is also meant for Android) but that Codemagic CLI tools are open source and can be used also locally or in other environments and read more about it in Using Codemagic CLI tools locally.